Composer/packages/client/src/store/action/skill.ts

```diff
@@ -36,6 +36,10 @@ export const updateSkillManifest: ActionCreator = ({ dispatch }, { content, id }
   });
 };
 
+export const checkSkillUrl = async (_store, { projectId, url }) => {
+  return await httpClient.post(`/projects/${projectId}/skill/check`, { projectId, url });
+};
```

Comment:
This is not a good usage of an action creator. I'll outline what I think should happen in another comment.